XML-decryption module not found

### Description:

Breaking bug: xml-encryption module not found on docker upon login

### Steps to reproduce:

1. Login via saml
2. fatal error:
```
Sorry, an annoying error occured
Error: Cannot find module 'xml-encryption'
Close Window
```
on docker
### Expected behavior:

No crash

### Actual behavior:


```
Sorry, an annoying error occured
Error: Cannot find module 'xml-encryption'
Close Window
```


### Server Setup Information:
```
rocketchat_1          | ➔ +-------------------------------------------------+
rocketchat_1          | ➔ |                  SERVER RUNNING                 |
rocketchat_1          | ➔ +-------------------------------------------------+
rocketchat_1          | ➔ |                                                 |
rocketchat_1          | ➔ |  Rocket.Chat Version: 0.74.0                    |
rocketchat_1          | ➔ |       NodeJS Version: 8.11.4 - x64              |
rocketchat_1          | ➔ |             Platform: linux                     |
rocketchat_1          | ➔ |         Process Port: 3000                      |
rocketchat_1          | ➔ |             Site URL: https://c***********  |
rocketchat_1          | ➔ |     ReplicaSet OpLog: Enabled                   |
rocketchat_1          | ➔ |          Commit Hash: cc5391e58c                |
rocketchat_1          | ➔ |        Commit Branch: HEAD                      |
rocketchat_1          | ➔ |                                                 |
rocketchat_1          | ➔ +-------------------------------------------------+
```


- Version of Rocket.Chat Server: 
- Operating System:  linux
- Deployment Method: docker
- Number of Running Instances: 1
- DB Replicaset Oplog: 
- NodeJS Version: 
- MongoDB Version: 

### Additional context

Login via simplesaml

### Relevant logs:

<!-- Logs from both SERVER and BROWSER -->
<!-- For more information about collecting logs please see: https://rocket.chat/docs/contributing/reporting-issues#gathering-logs -->

--- a/package.json
+++ b/package.json
@@ -201,6 +201,7 @@
 		"webdav": "^2.0.0",
 		"wolfy87-eventemitter": "^5.2.5",
 		"xml-crypto": "^1.0.2",
+		"xml-encryption": "0.11.2",
 		"xml2js": "0.4.19",
 		"xmlbuilder": "^10.1.1",
 		"xmldom": "^0.1.27",

